docker-compose.yml

```diff
@@ -7,7 +7,8 @@ services:
     - 8080:8080
     volumes:
     - .:/var/www/html # Non Mac users.
-    - ~/.composer:/var/www/.composer
+    - ~/.composer/cache:/var/www/.composer/cache
```

Comment:
The `cache folder` should be a personal override of docker-compose file, because it's only for performance.

A PR was created on `fpfis/httpd-php-dev` to configure the cache folder for compose to `/cache`, it will be available soon.

After that you should create a docker-compose.cache.yml with 
```
version: '2'
services:
  web:
    volumes:
      - <local_cache_folder>:/cache
```

and use the command 
`docker-compose -f docker-compose.yml -f docker-compose.cache.yml up -d`

for more information and to have a workaround for now, please see https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1515

Reply:
Thanks @voidtek , this makes sense, going to close this PR.